htdocs/compta/facture/class/facture.class.php

```diff
@@ -4230,19 +4230,6 @@ public function deleteline($rowid, $id = 0)
 
 		$this->db->begin();
 
-		// Free discount linked to invoice line
```

Comment:
I don't understand why you remove this ?

Reply:
Because I move it to delete method